src/Bpi/ApiBundle/Domain/Entity/UserFacet.php

```diff
@@ -0,0 +1,89 @@
+<?php
+namespace Bpi\ApiBundle\Domain\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+
+/**
+ * Bpi\ApiBundle\Domain\Entity\UserFacet
+ */
+class UserFacet
+{
+    /**
+     * @var MongoId $id
+     */
+    protected $id;
+
+    /**
+     * @var string $userId
+     */
+    protected $userId;
+
+    /**
+     * @var collection $facetData
+     */
+    protected $facetData;
+
+    /**
+     * Create a new UserFacet
+     */
+    public function __construct()
+    {
+        $this->facetData = new ArrayCollection();
```

Comment:
In UserFacetFepository class facetData parameter casted into stdObject with (object), but here it's declared as ArrayColection.